kino_webapp/config/dev.exs

```diff
@@ -7,7 +7,7 @@ use Mix.Config
 # watchers to your application. For example, we use it
 # with brunch.io to recompile .js and .css sources.
 config :kino_webapp, KinoWebapp.Endpoint,
-  http: [port: 4000],
+  http: [port: 4002],
```

Comment:
Did this since grafana runs on 4000, but now I think I'll make it the other way (run grafana / graphite on 4002 / 4003 and leave the webapp with defaults)